rdrf/rdrf/templates/admin/current_notification_table.html

```diff
@@ -0,0 +1,54 @@
+{% load i18n admin_static %}
```

Comment:
I like that you went the extra mile to create a user-friendly view on the existing notifications, but can't we just create filters (filter by registry) on the existing django admin Email Notifications list page and link to those?
That way we wouldn't duplicate existing pages...

Reply:
I wasn't aware of admin filters. I'm happy to use those if I can get them to work.